mantidproject/fitbenchmarking
# Added new directory structure

#### Description of Work

Fixes #77 
Added a new directory struncture in which functions are organized into modules and subdirectories. Prepared the way to extending the tool to use other minimizers outside mantid.

#### Testing Instructions

1. In the `mantidpython` terminal, run `/example_scripts/example_runScripts.py` .
2. Check if the output tables are as expected (i.e. produce the same results as master).

**Notes:**

* EVS14188-90_processed Gaussian peaks 1 visual display page does not work. This is a known issue, check #92 
* The functions lack descriptions in the current state. They will be added in #88 , as discussed.
